configs/build_config.py

```diff
@@ -155,6 +156,20 @@ def manifesting_keys(self):
     def tftp_keys(self):
         return frozenset(self._tftp_env)
 
+    @property
+    def golden_image(self):
+        path = self._settings['FILESYSTEM_IMAGES'] + '/golden/golden.*.tar'
```

Comment:
1. Doesn't this conflict with the change above where you removed the arch?
2. Check the regular expression compliance of glob.glob but I think you need
    '/golden/golden\..*\.tar'

Reply:
Kk, will double check that one